### PR TITLE
Text Analytics v2.1 preview

### PR DESCRIPTION
**Pull request by assafi**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

OAV is green.
Azure Validator throws a few transient warnings and one error regarding the versioning scheme. The version scheme of the service is following the OneAPI Azure guidelines, so safe to ignore. Same errors and warnings already exists in previous versions of the SDK.

### CODE REVIEW

**Review by yangyuan — APPROVED** [review]

**Comment by assafi** [comment]
@jianghaolu Could you please have a look at this PR? We want to move forward with the SDK update.

File: specification/cognitiveservices/data-plane/TextAnalytics/readme.md [hunk 1]
```diff
@@ -113,17 +113,17 @@ namespace ConsoleApp1
 
 > see https://aka.ms/autorest
 
-The current release is `release_2_0`.
+The current release is `release_2_1`.
```
Comment (jianghaolu) [inline specification/cognitiveservices/data-plane/TextAnalytics/readme.md]: Why do you remove release_2_0 considering 2.1 is still in preview?

Comment (assafi) [reply specification/cognitiveservices/data-plane/TextAnalytics/readme.md]: I'm not sure what's the correct value here. Am I suppose to add the new version side-by-side or leave only the non-preview version?

Comment (yangyuan) [reply specification/cognitiveservices/data-plane/TextAnalytics/readme.md]: @assafi is the 2.1 API in "preview" state?

@jianghaolu if 2.1 is a preview version, are you suggesting to leave `release_2_0` in configuration and add the extra tag for `2.1`?

**Pull request merged**